Lib/fontParts/base/info.py

```diff
@@ -7,16 +7,12 @@
 from fontParts.base.deprecated import DeprecatedInfo
 
 
-_copyAttributes = ()
-if isinstance(fontInfoAttributesVersion3, tuple):
-    _copyAttributes = fontInfoAttributesVersion3
-    _copyAttributes.remove("guidelines")
-    _copyAttributes = tuple(_copyAttributes)
-
 
 class BaseInfo(BaseObject, DeprecatedInfo):
 
-    copyAttributes = _copyAttributes
+    copyAttributes = fontInfoAttributesVersion3
+    copyAttributes.remove("guidelines")
```

Comment:
this is not good, this will also remove "guidelines" from the original `fontInfoAttributesVersion3`

Reply:
you can simple do `set(fontInfoAttributesVersion3)`